File: app/src/main/java/org/tasks/activities/attribution/LicenseRow.java
package org.tasks.activities.attribution;

import android.view.View;
import android.widget.TextView;
import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;
import butterknife.BindView;
import butterknife.ButterKnife;
import org.tasks.R;

class LicenseRow extends RecyclerView.ViewHolder {

  @BindView(R.id.copyright_holder)
  TextView copyrightHolder;

  @BindView(R.id.libraries)
  TextView libraries;

  LicenseRow(@NonNull View itemView) {
    super(itemView);
    ButterKnife.bind(this, itemView);
  }

  void bind(String copyrightHolder, String libraries) {
    this.copyrightHolder.setText(copyrightHolder);
    this.libraries.setText(libraries);
  }
}

File: app/src/main/java/org/tasks/activities/attribution/LicenseRow.kt
package org.tasks.activities.attribution

import android.view.View
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import butterknife.BindView
import butterknife.ButterKnife
import org.tasks.R

internal class LicenseRow(itemView: View) : RecyclerView.ViewHolder(itemView) {
    @BindView(R.id.copyright_holder)
    lateinit var copyrightHolder: TextView

    @BindView(R.id.libraries)
    lateinit var libraries: TextView

    init {
        ButterKnife.bind(this, itemView)
    }

    fun bind(copyrightHolder: String?, libraries: String?) {
        this.copyrightHolder.text = copyrightHolder
        this.libraries.text = libraries
    }
}